src/Orchard.Environment.Shell.Abstractions/Builders/ShellContext.cs

```diff
@@ -27,7 +27,7 @@ public class ShellContext : IDisposable
         /// </summary>
         public IServiceScope CreateServiceScope()
         {
-            return ServiceProvider.GetRequiredService<IServiceScopeFactory>().CreateScope();
+            return ServiceProvider.CreateScope();
         }
```

Comment:
Exactly, but maybe we still need the method to use the `ServiceProvider` of a given `ShellContext`.